app/src/main/java/com/infomaniak/mail/ui/main/InvalidPasswordFragment.kt

```diff
@@ -53,6 +54,7 @@ class InvalidPasswordFragment : Fragment() {
 
         enterPasswordDescription2.text = getStringWithBoldArg(R.string.enterPasswordDescription2, navigationArgs.mailboxEmail)
 
+        passwordInputLayout.fixIconPaddingIssue()
```

Comment:
Make a class that inherits from TextInputLayout and automatically applies the fix in its `init {}`